backend/app/methods/campaignUpdateMethods.js

```diff
@@ -0,0 +1,35 @@
+const { CampaignUpdate } = require('../models')
+
+function getCampaignUpdate(req, res) {
+  return CampaignUpdate
+    .findById(req.params.id)
+    .populate({
+      path: 'campaignCall',
+      populate: 'userActions committees'
+    })
+    .populate({
+      path: 'userConversations',
+      populate: {
+        path: 'user'
+      }
+    })
+    .exec()
+    .then(async function(campaignUpdate) {
+      return Object.assign(
+        {},
+        campaignUpdate.toObject(),
+        {
+          campaignCall: Object.assign(
+            campaignUpdate.campaignCall.toObject(),
+            { matchingRepresentatives: await campaignUpdate.campaignCall.getMatchingRepresentatives() }
```

Comment:
I added this issue for the future https://github.com/CallParty/CallParty/issues/228